### ISSUE TITLE
SC 3.2.6 Finding Help SC text

### ISSUE THREAD

**Issue by littlea11ywidget** [w3c/wcag#1314]
Feedback on the copy for the Success Criteria 3.2.6 Finding Help. 
I think the emphasis on 'one' can potentially make things confusing or at least it did for me. I kept wondering what the 'one option' was referring to. I could be misinterpreting it but I kept reading the mention of 'one' as 2 separate things and later realized it's the same. 

Would something to the effect of the below help to clarify? Again it could just be me misinterpreting the information. 

For single page Web applications or any set of Web pages, if one of the following is available, then it should it should be applied to each page in the same relative order: 

- Human contact details;
- Human contact mechanism;
- Self-help option;
- A fully automated contact mechanism.

If more than one option is available than a minimum of one must be provided.

**Comment by LaurenceRLewis** [comment]
@littlea11ywidget I agree the sentence is confusing. 

> if one of the following is available, then access to at least one option is included in the same relative order on each page:
> 
> Human contact details;
> Human contact mechanism;
> Self-help option;
> A fully automated contact mechanism.’
> 

Depending on the implied meaning I suggest the following change options. 

if one of the options **in the following list** is available, then access to at least one **other** option is included in the same relative order on each page. 

At least one option from the following list must be included in the same relative order on each page

If neither of the suggestions described the intended purpose, clarification of intent is needed.

**Comment by scha14** [comment]
Proposed changes : https://github.com/w3c/wcag/pull/1356, https://github.com/w3c/wcag/pull/1357

**Comment by guyhickling** [comment]
@littlea11ywidget, You are quite right about this, and I'm glad to see I'm not the only one to consider this SC very difficult for people to understand. Fortunately they are putting an amendment through for it now, so it will say:

> if one or more of the following help mechanisms is available

Does that make it clearer in your opinion? The link to see the amendment is https://github.com/w3c/wcag/pull/1357/files and https://github.com/w3c/wcag/pull/1357/files. (For full details see the latest post by @scha14 a few days ago on #1231.)

**Comment by alastc** [comment]
Hi everyone, we discussed this in a meeting and whilst agreeing with the substance of the comment, it's a hard terminology problem.

We tried: adding 'option' to both ends, and tried the terms mechanism, method, feature, means and ways. Each of these, particularly mechanism/method/feature imply things we do not intend.

We'd like to propose going with this as the first part of the SC text:
> For single page Web applications or any set of Web pages, if one or more of the following ways of finding help is provided, then access to at least one way of finding help is included in the same relative order on each page:

Also given the discussion here, does resolve the issue you raised?

**Comment by scha14** [comment]
Closing based on discussion in https://github.com/w3c/wcag/issues/1231

Suggestion about including **appearing** is included in the discussion that combines all 3.2.6 issues https://github.com/w3c/wcag/issues/1242

Please reopen if this remains unaddressed. Thank you!
